lib/client-assets.php

```diff
@@ -66,6 +66,14 @@ function gutenberg_get_script_polyfill( $tests ) {
 	return $polyfill;
 }
 
+
+function gutenberg_script_debug() {
+	return (
+		( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ||
+		file_exists( gutenberg_dir_path() . '.git' )
```

Comment:
What if, in the course of development, I do want to test with minified souces? Switching on the existence of `.git` seems like a very inflexible method.

Reply:
An environment variable seems the best option here; why not have `MINIFY_JS`, `SCRIPT_DEBUG`, or something else, be an env var we could check for?